### ISSUE TITLE
Accommodate the nanopore (only) and hybrid datasets and reflect on the final results (nanopore/illumina)

### ISSUE THREAD

**Issue by abhi18av** [TORCH-Consortium/MAGMA#150]

**Comment by TimHHH** [comment]
> @vrennie @TimHHH , where exactly do we need to add this sequencing platform information i.e. which summary files?

I would think a column in the summary stats file.

**Comment by vrennie** [comment]
Yes, I agree with Tim, just a column that looks like this:

**Sequencing Technology** 
Illlumina
ONT
ONT
Illumina
Illumina
Illumina
...

**Comment by abhi18av** [comment]
Okay, I understand this would be added to the `summary stats file` 👍 

However, there's one more detail worth mentioning here, currently we hard-code the sequencing technology in the `bam_rg_string`  https://github.com/TORCH-Consortium/MAGMA/blob/786d13dfe1988784f870499bb878f47cc945a493/workflows/validate_fastqs_wf.nf#L30 

Should we not add this column to the input-samplesheet as well?

**Comment by vrennie** [comment]
Yes, good catch @abhi18av, lets add this as a column to the samplesheet.

**Comment by TimHHH** [comment]
Yes, ideally the user provides the sequencing technology in the sample sheet and this is then used in the ```bam_rg_string``` along the lines of ```${technology}```

**Comment by abhi18av** [comment]
Guys, what about reflecting that on the actual sample name as well? Something like `Shea2017_2021_396.SRR16089406.LNA.A1.ILMN.1.1.1`

The NCBI currently lists the following _platforms_ used for the sequences

- ILLUMINA   
- ION_TORRENT   
- ABI_SOLID   
- PACBIO_SMRT   
- CAPILLARY   
- OXFORD_NANOPORE   
- LS454   
- BGISEQ

To avoid long names, we can perhaps standardize the acronyms like `ILMN` / `ONT` / `PCB` / `ION` etc - what do you think?

**Comment by vrennie** [comment]
I think unless the full name messes up the .csv its better to keep the full name
